fluent-package/Gemfile

```diff
@@ -88,15 +88,15 @@ gem "opensearch-ruby", "3.4.0"
 gem "fluent-plugin-opensearch", "1.1.5"
 gem "fluent-plugin-fluent-package-update-notifier", "0.2.3"
 gem "fluent-plugin-obsolete-plugins", "0.2.2"
-gem "fluent-plugin-opentelemetry", "0.3.0"
+gem "fluent-plugin-opentelemetry", "0.4.0"
```

Comment:
Minor update.
It just added new `in_opentelemetry_metrics` plugin, 
it has no effect whatsoever on past versions.

https://github.com/fluent-plugins-nursery/fluent-plugin-opentelemetry/blob/main/CHANGELOG.md#040---2025-10-10